### ISSUE TITLE
Create Tasks from Geotagged Photos

### ISSUE THREAD

**Issue by mvexel** [osmlab/maproulette3#1759]
Inspired by https://twitter.com/mvexel/status/1532114607120130049 

This is a big one and needs quite a bit of thinking and discussion—and work.

To create a 'photo challenge' you would point MapRoulette to an place (online / local?) that contains a set of geotagged images. MapRoulette would create a task from each image. There might be a special widget to display the image inline on the task page, or just on the map like JOSM does.

**Comment by mvexel** [comment]
Perhaps this could be done outside of MapRoulette, using https://github.com/osmlab/maproulette-python-client @mattmanley ...?

**Comment by mvexel** [comment]
Some work ongoing here: https://github.com/mvexel/maproulette-photo-challenge

**Comment by joostschouppe** [comment]
As a more generic feature (that might be needed to make this one work), it'd be nice if you could include images in new tasks. That would be really convenient in my [Traffic sign project](https://github.com/osmbe/traffic-sign-project) task.

**Comment by mvexel** [comment]
@joostschouppe You can already include links in the task instructions, this includes links to externally hosted images, using the regular `![alt](url)` markdown syntax.

**Comment by mvexel** [comment]
@mattmanley created a PR that has a rough working implementation of the Python tool to create this type of challenge: https://github.com/mvexel/maproulette-photo-challenge/pull/3

**Comment by joostschouppe** [comment]
Martijn, the issue isn't the hosting of the files. The issue is that there are loads and loads of different traffic signs. So it'd be nice to show an image of the new traffic sign instead of just the code.

**Comment by mvexel** [comment]
@joostschouppe right, for that you would use the [`![]()` syntax to show an image hosted elsewhere inline](https://www.markdownguide.org/basic-syntax/#images-1) in the task instruction. Or is it something else you want to achieve?

**Comment by mvexel** [comment]
> @joostschouppe right, for that you would use the [`![]()` syntax to show an image hosted elsewhere inline](https://www.markdownguide.org/basic-syntax/#images-1) in the task instruction. Or is it something else you want to achieve?

There is a ticket over on the docs repo to clarify that you can use image links: https://github.com/maproulette/docs/issues/22

**Comment by mvexel** [comment]
No problem @joostschouppe . Let me know if you have any problems with this. I haven't actually tested it myself, but the markdown parser should support setting image display size [like this](https://www.markdownguide.org/hacks/#image-size).
